Packages/MIES/MIES_IgorHooks.ipf

```diff
@@ -191,3 +191,26 @@ static Function AfterCompiledHook()
 End
 
 #endif
+
+/// @brief This function is called after an experiment is opened. It implements a forced synchronization between the
+/// GUIstate waves and the DA_Ephys panel if a panel was recreated by the DA_Ephys macro on experiment load.
+/// The synchronization is enforced by deleting the GUIState waves. This synchronized the GUI state from the controls
+/// when the next control content is changed.
+/// AfterCompileHook is one of Igors predefined integrated hook functions, see Help file: Advanced Topics.ihf (Topic: AfterFileOpenHook)
+/// @param[in] refNum         reference number of the file
+/// @param[in] fileNameStr    name of file
+/// @param[in] pathNameStr    path to file
+/// @param[in] fileTypeStr    macintosh file type code
+/// @param[in] fileCreatorStr macintosh creator code
+/// @param[in] fileKind       kind of file code
+static Function AfterFileOpenHook(refNum, fileNameStr, pathNameStr, fileTypeStr, fileCreatorStr, fileKind)
+	variable refNum
+	string fileNameStr, pathNameStr, fileTypeStr, fileCreatorStr
+	variable fileKind
+
+	string list = WinList(BASE_WINDOW_TITLE, ";", "WIN:64")
+	if(!isEmpty(list))
+		KillOrMoveToTrash(wv = GetDA_EphysGuiStateNum(BASE_WINDOW_TITLE))
+		KillOrMoveToTrash(wv = GetDA_EphysGuiStateTxT(BASE_WINDOW_TITLE))
+	endif
+End
```

Comment:
@MichaelHuth I think this is not required. We call `DAP_UnlockDevice` for all locked devices via `IH_Cleanup`, and that is called before closing and experiment. This should catch the same case you want to catch here or?

Reply:
@t-b I don't agree, consider this case: 

Dev1 is locked, you save the experiment and reload it, what happens is:
- IH_Cleanup unlocks "Dev1", thus deletes the GUIState waves for Dev1.
- On loading the saved experiment "DA_Ephys" is created from Macro and these GUIState waves were not deleted on Unlock.

We could delete the GUIState waves as well in `DAP_UnlockDevice` for `panelTitleUnlocked` which is `BASE_WINDOW_TITLE`. This would work in the way you propose. What do you think?

Reply:
@MichaelHuth 

> We could delete the GUIState waves as well in DAP_UnlockDevice for panelTitleUnlocked which is BASE_WINDOW_TITLE. This would work in the way you propose. What do you think?

Sounds good.